app/src/main/java/org/simple/clinic/summary/PatientSummaryConfig.kt

```diff
@@ -1,6 +1,25 @@
 package org.simple.clinic.summary
 
+import io.reactivex.Observable
+import org.simple.clinic.remoteconfig.ConfigReader
+import org.threeten.bp.Duration
+
 data class PatientSummaryConfig(
     val numberOfBpPlaceholders: Int,
-    val numberOfBpsToDisplay: Int
-)
+    val numberOfBpsToDisplay: Int,
+    val bpEditableDuration: Duration
+) {
+
+  companion object {
+
+    fun read(configReader: ConfigReader): Observable<PatientSummaryConfig> {
+      val bpEditableDurationConfig = configReader.long("bp_editable_duration_in_minutes", 60)
```

Comment:
This would mean that we won't be able to change the editable duration to less than a minute remotely. Should we make this represent seconds instead of minutes?

Reply:
That's a good point.
I will make this change.